src/components/AutoAdvanceToggle/AutoAdvanceToggle.test.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { fireEvent, render, screen } from '@testing-library/react';
+import AutoAdvanceToggle from "./AutoAdvanceToggle";
+import manifest from '@TestData/lunchroom-manners';
+import autoAdvanceManifest from '@TestData/multiple-canvas-auto-advance';
+import { withManifestProvider } from '../../services/testing-helpers';
+import * as utils from '@Services/utility-helpers';
+
+describe('AutoAdvanceToggle', () => {
+  describe('with manifest without auto-advance behavior', () => {
+    beforeEach(() => {
+      const NonAutoManifest = withManifestProvider(AutoAdvanceToggle, {
+        initialState: { manifest, canvasIndex: 0 },
+      });
+      render(<NonAutoManifest />);
+    });
+    test('renders auto advance toggle successfully', () => {
+      expect(screen.queryByTestId('auto-advance')).toBeInTheDocument();
+      expect(screen.queryByTestId('auto-advance-label')).toBeInTheDocument();
+      expect(screen.queryByTestId('auto-advance-toggle')).toBeInTheDocument();
+    });
+    test('auto advance is disabled by defulat', () => {
+      const toggle = screen.queryByTestId('auto-advance-toggle');
```

Comment:
I tried DRYing this up but couldn't find any pattern that works.

Reply:
This could be done by initializing `toggle` before the `beforeEach` block and then assigning the value inside the `beforeEach` block.
Example: https://github.com/avalonmediasystem/react-structural-metadata-editor/blob/7fec5982340fda3a49058b85ba47c40363d18371/src/components/__tests__/ListItemControls.test.js#L124